main.go

```diff
@@ -8,6 +8,7 @@ import (
 	"errors"
 	"flag"
 	"fmt"
+	"github.com/observatorium/api/instrumentation"
```

Comment:
a nit but for this and others, we might want to retain the existing organisation of imports

Reply:
Ah, the order of imports. Let me fix my IDE config. 😂

Reply:
Yup, goimports is needed. Raised relevant PR: https://github.com/observatorium/api/pull/431